src/index.js

```diff
@@ -21,6 +21,8 @@ export const callWaiter = actions.callWaiter;
 export const clearWaiter = actions.clearWaiter;
 export const prepareWaiter = actions.prepareWaiter;
 export const destroyWaiter = actions.destroyWaiter;
+export const clearAll = actions.clearAll;
+export const destroyAll = actions.destroyAll;
```

Comment:
Not sure if you're interested, but I think we could have some more concise syntax here with destructuring:
```
export const {
  callWaiter,
  clearWaiter,
  prepareWaiter,
  destroyWaiter,
  clearAll,
  destroyAll,
} = actions
```
Just an alternative :)

Reply:
sounds good 👍